build.gradle

```diff
@@ -161,9 +161,9 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.8.0',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:[4.1.4.Final]',
-                netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.4.Final' + epoll_suffix,
-                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork19',
+                netty: 'io.netty:netty-codec-http2:[4.1.6.Final]',
+                netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.6.Final' + epoll_suffix,
+                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork23',
```

Comment:
@ejona86 I am not sure whether that is necessary from a security perspective. I am also not sure where to check. Skimming over the netty-tcnative commits, it looks like openssl was updated to version 1.0.2j ... not sure if that's useful :) Maybe @nmittler knows :)